src/wallet/rpcwallet.cpp

```diff
@@ -2565,12 +2565,15 @@ UniValue z_listunspent(const UniValue& params, bool fHelp)
             obj.push_back(Pair("jsindex", (int)entry.jsop.js ));
             obj.push_back(Pair("jsoutindex", (int)entry.jsop.n));
             obj.push_back(Pair("confirmations", entry.nHeight));
-            obj.push_back(Pair("spendable", pwalletMain->HaveSproutSpendingKey(boost::get<libzcash::SproutPaymentAddress>(entry.address))));
+            bool hasSproutSpendingKey = pwalletMain->HaveSproutSpendingKey(boost::get<libzcash::SproutPaymentAddress>(entry.address));
```

Comment:
Minor suggestion: rename this haveSproutSpendingKey to match the name of the method from which it's derived.

Reply:
I wrote the variable as 'has' because I think that is more grammatical, but I didn't think it worth it to go through the trouble of changing the method 'Have...'.